src/browser.js

```diff
@@ -10,10 +10,68 @@ exports.formatArgs = formatArgs;
 exports.save = save;
 exports.load = load;
 exports.useColors = useColors;
-exports.storage = 'undefined' != typeof chrome
-               && 'undefined' != typeof chrome.storage
-                  ? chrome.storage.local
-                  : localstorage();
+
+var isFirefox = false;
+var isFirefoxNewerThan31 = false;
+var isAppleWebkit = false;
+
+if (typeof navigator !== 'undefined' &&
+    typeof navigator.userAgent !== 'undefined') {
+  isFirefox = navigator.userAgent.toLowerCase().match(/firefox\/(\d+)/)
```

Comment:
This check is a bit wonky for me because I am using [the random agent spoofer extension][1]. Suddenly it stopped working for no particular reason, as it randomizes the agent every now and then.

Can we find a better way to check than this?

[1]: https://addons.mozilla.org/en-US/firefox/addon/random-agent-spoofer/